lighthouse-core/audits/metrics/first-contentful-paint.js

```diff
@@ -32,21 +32,21 @@ class FirstContentfulPaint extends Audit {
   }
 
   /**
-   * @return {{mobile: LH.Audit.ScoreOptions, desktop: LH.Audit.ScoreOptions}}
+   * @return {{mobileScoring: LH.Audit.ScoreOptions, desktopScoring: LH.Audit.ScoreOptions}}
    */
   static get defaultOptions() {
     return {
-      mobile: {
-        // 75th and 95th percentiles HTTPArchive -> median and PODR
+      mobileScoring: {
```

Comment:
#9911 changed these for 6.0, so we haven't released the breaking change for them yet, so we still have some time to tweak. Open to bike shedding :)

Reply:
I know @paulirish just said he liked this change, but I liked the idea that broadly supported options bifurcated by device form factor beyond scoring.

While most of these are just scoring I did *just* raise the point that it would make sense to adjust some of our other thresholds in other diagnostic audits based on mobile/desktop, not just scoring and it would be weird to see 

`mobileScoring`
`desktopScoring`
`mobileIgnoreThreshold`
`desktopIgnoreThreshold` 

instead of just

`options.mobile` and `options.desktop` are both typed the exact same and are the full set of defined options

Reply:
This is what happens when `options` are an untyped grab bag of anything goes and no one writes down what the goals are :P

I was mostly avoiding repetition (`{median: options.scoreMedian, p10: options.scoreP10}` or whatever) because nothing else was using these but that's fine.

Reply:
@patrickhulce what do you think of this options approach

Reply:
love it! 👍  🎉